spec/models/call_off_contract_spec.rb

```diff
@@ -8,8 +8,16 @@
   describe "associations" do
     it { is_expected.to have_many(:participant_bands) }
 
-    it "is expected to have band_a with nil as the lowest min value" do
-      expect(call_off_contract.band_a.min).to eq(nil)
+    it "is expected to have band_a with the lowest minimum value" do
+      expect(call_off_contract.band_a.min.to_i).to eq(0)
+    end
+
+    it "is expected to have a total contract value" do
+      expect(call_off_contract.total_contract_value).to eq(2000 * 995) # recruitment_target * per_participant
+    end
```

Comment:
Are these the default values from our test suite? Maybe we should do something like `call_off_contract.recruitment_target * call_off_contract.per_participant`? Rather than adding a comment about it?

Reply:
I did that originally, but then it's just testing both side the same.